gems/pending/VMwareWebService/VimTypes.rb

```diff
@@ -95,6 +103,10 @@ def vimType=(val)
     @vimType = val.nil? ? nil : val.to_sym
   end
 
+  def vimBaseType
```

Comment:
Why move them to a mixin, when they are specific to this class, and wouldn't be mixed into any other?